subprojects/core/src/main/java/org/gradle/api/internal/project/taskfactory/InputPropertyAnnotationHandler.java

```diff
@@ -36,9 +37,9 @@ public void update(TaskInternal task, Callable<Object> futureValue) {
         });
         Class<?> valueType = context.getValueType();
         if (File.class.isAssignableFrom(valueType)
+            || java.nio.file.Path.class.isAssignableFrom(valueType)
```

Comment:
I guess you are not using an import to avoid warnings in the file?

Reply:
Yeah.